web/src/beta/features/Assets/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const fileFormats = ".kml,.czml,.topojson,.geojson,.json,.gltf,.glb";
+
+export const imageFormats = ".jpg,.jpeg,.png,.gif,.svg,.tiff,.webp";
```

Comment:
Can you UPDATE_CASE these? 
For constants in beta I am trying to do them all in uppercase to better differentiate them from other more local/fluid values 🙏🏻

Reply:
note that in Beatrice pr, forgot to ask about that, thank you for mention